book/_toc.yml

```diff
@@ -27,19 +27,20 @@
       sections:
         - file: tutorials/day_1
           sections:
-            - file: tutorials/jupyter
-            - file: tutorials/core-datasets/index
-              sections:
-                - file: tutorials/core-datasets/01_introduction
-                - file: tutorials/core-datasets/02_data-package
-                - file: tutorials/core-datasets/03_practice-querying
```

Comment:
@asteiker - This looks like a faulty merge and the yml the indentation needs to remain unchanged for day 1

Reply:
@jomey I _think_ I fixed the indents. Let me know if you still see anything amiss.

Reply:
Hm .. Sorry to be a pain here and yml can be challenging.

Child entries under a section should be followed with two spaces:
```yml
- file: A
  sections:
    - file B:
      sections:
```
Currently I see four under the section for day 1 and day 2